### DISCUSSION TITLE
How can I render a UI to a texture, and still interact with it? (or create a similar effect)

### DISCUSSION THREAD

**Discussion by Teajey** [bevyengine/bevy#5712] [Q&A]
I want to be able to resize the game screen uniformly to the window as demonstrated here:

https://user-images.githubusercontent.com/21069848/184854308-c06adf60-0f54-48d7-a2cc-6a7569abbbc6.mov

I rendered the game screen, including UI, to a sprite texture so that I could achieve this. But I've found that the UI becomes non-interactable this way. I understand that this is because Bevy only really supports one UI tree in it's current state, on the default window render target.

Is it necessary to create an issue for this? 

Is there an alternative way to achieve the same effect? I'd prefer not to have the screen resize disproportionately with flexbox and having to update the size of all entities on screen, etc.

Thanks for any help!

**Comment by KirmesBude** [comment]
You might be able to achieve something like this using camera Viewport and setting physical_position/physical_size based on window size and your desired aspect ratio.
https://github.com/bevyengine/bevy/blob/main/examples/3d/split_screen.rs

**Reply by Teajey** [reply]
This isn't what I'm after. Notice how the viewports clip the edges of the fox when you resize the window. Whereas in my above example the entire screen resizes uniformly and proportionally. Also, I like rendering to an image to get a lower pixel density. I'm more looking for a way to override the UI interaction system or something

**Comment by SET001** [comment]
Faced same issue here. @Teajey have you found any solution?

**Reply by Teajey** [reply]
Unfortunately, no. I've put my project on ice until Bevy has proper support...

**Reply by Teajey** [reply]
Here's two issues that I'm tracking: https://github.com/bevyengine/bevy/issues/5622 https://github.com/bevyengine/bevy/issues/5570
